### ISSUE TITLE
nltt_diff_exact cannot handle trees with stem

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/nLTT#24]
`nltt_diff_exact`'s nephew `nltt_diff`, works great:

```
phylogeny_1 <- ape::read.tree(text = "(a:1,b:1):1;")
phylogeny_2 <- ape::read.tree(
  text = "(((d:0.000000001,c:0.000000001):1,b:1):0.000000001,a:1.0):1;")
nltt_stat <- nLTT::nltt_diff(phylogeny_1, phylogeny_2)
testthat::expect_equal(nltt_stat, 0.25, tolerance = 0.0001)
```

nltt_diff_exact, however, fails:

```
nltt_stat <- nLTT::nltt_diff_exact(phylogeny_1, phylogeny_2)
```

with the message:
```
Error: all(b_times2 <= 0) is not TRUE
Called from: nltt_diff_exact_brts(b_times = b_times, lineages = lineages, 
    b_times2 = b_times2, lineages2 = lineages2, distance_method = distance_method)
```

**Comment by richelbilderbeek** [comment]
There is a problem with the tree I supplied, as `ape::branching.times` converts one branching time to negative. Use this instead:

```
  tree2 <- ape::read.tree(
    text = "(((d:0.000000001,c:0.000000001):1,b:1):0.000000001,a:1.000000001):1;")
```

**Comment by richelbilderbeek** [comment]
Fixed this witj #25
